src/main/resources/assets/paths/findAllStudyNames.yaml

```diff
@@ -0,0 +1,16 @@
+get:
+  summary: Find All Study Names
+  description: Returns a list of all the actively used study names in the backend.
+  tags:
+    - Dataset
+  responses:
+    200:
+      description: dataset id
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              type: string
+    404:
+      description: Dataset not found
```

Comment:
I'm not clear on why we'd return a 404 from this API. I can see the need for a 500 in the case of a database error.